packages/instrumentation/src/ActiveTracingHelper.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Context,
+  context as _context,
+  ROOT_CONTEXT,
+  Span,
+  SpanContext,
+  SpanKind,
+  trace,
+  TraceFlags,
+} from '@opentelemetry/api'
+import { Span as SpanConstructor, Tracer } from '@opentelemetry/sdk-trace-base'
+import { EngineSpanEvent, ExtendedSpanOptions, SpanCallback, TracingHelper } from '@prisma/internals'
+
+// If true, will publish internal spans as well
+const showAllTraces = process.env.PRISMA_SHOW_ALL_TRACES === 'true'
+
+// https://www.w3.org/TR/trace-context/#examples-of-http-traceparent-headers
+// If traceparent ends with -00 this trace will not be sampled
+// the query engine needs the `10` for the span and trace id otherwise it does not parse this
+const nonSampledTraceParent = `00-10-10-00`
+
+export class ActiveTracingHelper implements TracingHelper {
+  constructor(private traceMiddleware: boolean) {}
```

Comment:
Could we make this an "options" object? As we might want to add more along middleware in the future.